flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/typeutils/AvroSchemaConverterTest.java

```diff
@@ -104,48 +104,115 @@ public void testRowTypeAvroSchemaConversion() {
 				DataTypes.FIELD("row3", DataTypes.ROW(DataTypes.FIELD("c", DataTypes.STRING())))))
 			.build().toRowDataType().getLogicalType();
 		Schema schema = AvroSchemaConverter.convertToSchema(rowType);
-		assertEquals("{\n" +
+		assertEquals("[ {\n" +
```

Comment:
I think it would be nice to add a test that we can convert back and forth between `DataType` and `Schema` in respect to the field names.

Reply:
Already added, see `AvroSchemaConverterTest.testConversionIntegralityNullable`.

Reply:
True, you test though only the `DataType` -> `Schema` and back. We could also add a test for `Schema` -> `DataType` (you start with a Schema), which is important in case of a schema registry.

Reply:
Thanks, i would add `schema` -> `DataType` -> `schema` test.